src/MDtoHTML.py

```
@@ -1,4 +1,94 @@
+        elif block_type == BlockType.ORDERED_LIST:
+            children_nodes.append(process_ol(process_block(block)))
+        elif block_type == BlockType.UNORDERED_LIST:
+            children_nodes.append(process_ul(process_block(block)))
+        elif block_type == BlockType.CODE:
+            block = re.sub(r'^```(\w+)?\n?', '', block)   # remove opening ```
+            block = re.sub(r'\n?```$', '', block)         # remove closing ```
+            block = textwrap.dedent(block)
+            children_nodes.append(process_code(block))
+    return ParentNode('div', children_nodes)
+
+def process_block(block: str) -> HTMLNode:
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)